### PR TITLE
Add const where possible

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: tests/fuzz/fuzz_compress.c [hunk 1]
```diff
@@ -8,7 +8,7 @@ extern "C" {
 #endif
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
```
Comment (kalvdans) [inline tests/fuzz/fuzz_compress.c:10]:
```suggestion
int LLVMFuzzerTestOneInput(const uint8_t *const data, const size_t size) {
```
Add const to the parameters also while you are at it. (These doesn't change the signature, just make them immutable in the function body)

Personally I think it is silly to add const to local variables, if it aids readability then function is too big :D

**Review by kalvdans — APPROVED** [review]

File: tests/fuzz/fuzz_compress.c (hunk 1 above)
Comment (DimitriPapadopoulos) [reply tests/fuzz/fuzz_compress.c:10]: The variable is passed by value, so `const` doesn't really help.

Also, the required signature is documented here:
https://llvm.org/docs/LibFuzzer.html#fuzz-target

Comment (kalvdans) [reply tests/fuzz/fuzz_compress.c:10]: `const` helps to not inadvertly modify the local copy inside the function body. It doesn't change the function signature.

**Comment by FrancescAlted** [comment]
Thanks @DimitriPapadopoulos

**Pull request merged**